tools/Utilities/EnvVariablesUtility/EnvironmentVariablesAppMainWindow.xaml.cs

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+using DevHome.EnvironmentVariables.Win32;
+using DevHome.Telemetry;
+using EnvironmentVariablesUILib;
+using EnvironmentVariablesUILib.Helpers;
+using EnvironmentVariablesUILib.ViewModels;
+using Microsoft.UI.Dispatching;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Serilog;
+using WinUIEx;
+
+namespace DevHome.EnvironmentVariables;
+
+public sealed partial class EnvironmentVariablesMainWindow : WindowEx
+{
+    private EnvironmentVariablesMainPage EnvVariablesUtilityMainPage { get; }
+
+    private string UtilityTitle { get; set; }
+
+    private readonly Guid activityId;
+
+    private readonly Serilog.ILogger _log = Log.ForContext("SourceContext", nameof(EnvironmentVariablesApp));
+
+    public EnvironmentVariablesMainWindow(Guid activityId)
+    {
+        this.activityId = activityId;
+        this.InitializeComponent();
+
+        Activated += MainWindow_Activated;
+
+        ExtendsContentIntoTitleBar = true;
+        SetTitleBar(AppTitleBar);
+
+        var loader = new Microsoft.Windows.ApplicationModel.Resources.ResourceLoader("PowerToys.EnvironmentVariablesUILib.pri", "PowerToys.EnvironmentVariablesUILib/Resources");
+        var title = EnvironmentVariablesApp.GetService<IElevationHelper>().IsElevated ? loader.GetString("WindowAdminTitle") : loader.GetString("WindowTitle");
+        UtilityTitle = title;
+
+        Title = UtilityTitle;
+        AppWindow.SetIcon(Path.Combine(AppContext.BaseDirectory, "Assets/EnvironmentVariables/EnvironmentVariables.ico"));
+
+        var handle = this.GetWindowHandle();
+        RegisterWindow(handle);
+
+        EnvVariablesUtilityMainPage = EnvironmentVariablesApp.GetService<EnvironmentVariablesMainPage>();
+
+        TelemetryFactory.Get<ITelemetry>().Log("EnvironmentVariablesApp_EnvironmentVariablesMainWindow_Initialized", LogLevel.Measure, new EmptyEvent(), this.activityId);
+        _log.Information("EnvironmentVariablesApp MainWindow Intialized");
+    }
+
+    private void Grid_Loaded(object sender, RoutedEventArgs e)
+    {
+        MainGrid.Children.Add(EnvVariablesUtilityMainPage);
+        Grid.SetRow(EnvVariablesUtilityMainPage, 1);
+
+        TelemetryFactory.Get<ITelemetry>().Log("EnvironmentVariablesApp_EnvironmentVariablesMainWindow_GridLoaded", LogLevel.Measure, new EmptyEvent(), this.activityId);
+        _log.Information("EnvironmentVariablesApp MainWindow Grid loaded");
+    }
+
+    private void MainWindow_Activated(object sender, WindowActivatedEventArgs args)
+    {
+        AppTitleBar.IsActive = args.WindowActivationState != WindowActivationState.Deactivated;
+    }
+
+    private static readonly DispatcherQueue _dispatcherQueue = DispatcherQueue.GetForCurrentThread();

```

Comment:
The tool tells me this doesn't have references anymore? Can it be deleted?

Reply:
Removed.